dvc/tree/base.py

```diff
@@ -202,6 +204,9 @@ def walk_files(self, path_info, **kwargs):
         """
         raise NotImplementedError
 
+    def ls(self, path_info, recursive=False, detail=False):
+        raise RemoteActionNotImplemented("ls", self.scheme)
```

Comment:
Let's try match fsspec's ls as much as we can. `recursive` is our extension supported only by some clouds.
```suggestion
    def ls(self, path_info, detail=False, **kwargs):
        raise RemoteActionNotImplemented("ls", self.scheme)
```